packages/ngx-forms/src/lib/image-select/components/image-select/image-select.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="grid-container">
```

Comment:
Voor CSS classes graag de guidelines lezen. In het kort: wij hanteren atomic BEM classes. Voor deze component zou een goede benaming bv. zijn: .o-image-select. Andere classes zijn dan bv. .o-image-select__container en .o-image-select__choice is-selected.

Reply:
Door gebruik te maken van een checkbox input element zijn deze custom classes niet meer nodig. Css is nu getarget op de standaard input selectors en events.